source/reference/minio-mc/mc-ready.rst

```diff
@@ -0,0 +1,106 @@
+============
+``mc ready``
+============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc ready
+
+Syntax
+------
+
+.. start-mc-ready-desc
+
+The :mc:`mc ready` command checks the status of a cluster.
+
+.. end-mc-ready-desc
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following sends a ``GET`` request to the cluster and returns its status.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc ready play
+
+      The command pings the deployment at the :mc:`~mc alias` ``play``.'
```

Comment:
`ping` and `GET` are somewhat distinct from a networking perspective.

And we already state it sends a GET request. So I'd just drop line 33

```suggestion
```

Reply:
Yes. Good catch. I did change that somewhere else before I did the PR, so glad you caught it here.